bubble-wrap-style.yaml

```diff
@@ -2806,31 +2817,430 @@ layers:
+                                /^US:UT$/.test(feature.network) ||
+                                /^US:VA$/.test(feature.network) ||
+                                /*TODO: US:VA:secondary*/
+                                /^US:VT$/.test(feature.network) ||
+                                /^US:WA$/.test(feature.network) ||
+                                /^US:WI$/.test(feature.network) ||
+                                /^US:WY$/.test(feature.network)
+                                /*US:CA*/
+                                /*US:I*/
+                                /*US:NY*/
+                                /*US:PA*/
+                                /*US:US*/
+                                /*county_shield*/
+                        }
+                    draw:
+                        mapzen_icon_library:
+                            texture: mapzen_icon_library_shields_usa
+                            sprite: |
+                                function() { return ( feature.network + '-' + feature.shield_text.length + 'char'); }
+                            text:
+                                # offset: [0px, 1px]
+                                font:
+                                    fill: [0.308,0.308,0.308]
+                                    #size: [[7,7px],[13,9px],[15,10px]]
+#                    width_1char:
+#                        filter: function() { return (feature.shield_text.length === 1) }
+#                        draw:
+#                            mapzen_icon_library:
+#                                size: [[6,[13px,17px]],[13,[15px,19px]],[15,[18px,22px]]]
+#                    width_2char:
+#                        filter: function() { return (feature.shield_text.length === 2) }
+#                        draw:
+#                            mapzen_icon_library:
+#                                size: [[6,[17px,17px]],[13,[19px,19px]],[15,[22px,22px]]]
+#                    width_3char:
+#                        filter: function() { return (feature.shield_text.length === 3) }
+#                        draw:
+#                            mapzen_icon_library:
+#                                size: [[6,[20px,17px]],[13,[22px,19px]],[15,[27px,22px]]]
+#                    width_4char:
+#                        filter: function() { return (feature.shield_text.length === 4) }
+#                        draw:
+#                            mapzen_icon_library:
+#                                size: [[6,[21px,17px]],[13,[24px,19px]],[15,[29px,22px]]]
+#                    width_5char:
+#                        filter: function() { return (feature.shield_text.length === 5) }
+#                        draw:
+#                            mapzen_icon_library:
+#                                size: [[6,[22px,17px]],[13,[25px,19px]],[15,[30px,22px]]]
+
+                international-shields:
+                    filter: |
+                        function() {
+                            return feature.shield_text &&
```

Comment:
You need to make the precedence clearer here, cases where `feature.shield_text` is null are getting through. Should be:

```
return feature.shield_text && (
  /^AM:AM.*$/.test(feature.network) ||
   ...
  /^ZA:R.*$/.test(feature.network)
 )
```

Reply:
fixed in https://github.com/tangrams/bubble-wrap/pull/262/commits/33743924702df949c53fee76bf59e38c4d1ad099